doc/source/getting-started.rst

```diff
@@ -0,0 +1,194 @@
+.. _getting-started:
+
+*********************
+Getting Started Guide
+*********************
+
+The following guide will step through an example of developing and testing a
+new Ansible role. After reading this guide, you should be familiar with the
+basics of how to use Molecule and what it can offer.
+
+.. note::
+
+    In order to complete this guide by hand, you will need to additionally
+    install `Docker`_.
+
+.. _Docker: https://docs.docker.com/
+
+Creating a new role
+-------------------
+
+Molecule uses `Ansible Galaxy`_ under the hood to generate conventional role
+layouts. If you've ever worked with roles before, you'll be right at home. If
+not, please review the `Role directory structure`_ guide to see what each
+folder is responsible for.
+
+.. _Ansible Galaxy: https://docs.ansible.com/ansible/latest/reference_appendices/galaxy.html#the-command-line-tool
+.. _Role directory structure: https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#role-directory-structure
+
+To generate a new role with Molecule, simply run:
+
+.. code-block:: bash
+
+    $ molecule init role -r my-new-role
+
+You should then see a ``my-new-role`` folder in your current directory.
+
+Molecule Scenarios
+------------------
+
+You will notice one new folder which is the ``molecule`` folder.
+
+In this folder is a single :ref:`root_scenario` called ``default``.
+
+Scenarios are the starting point for a lot of powerful functionality that
+Molecule offers. For now, we can think of a Scenario as a test suite for your
+newly created role. You can have as many scenarios as you like and Molecule
+will run one after the other.
+
+The Scenario Layout
+-------------------
+
+Within the ``molecule/default`` folder, we find a number of files:
+
+.. code-block:: bash
+
+    $ ls
+    Dockerfile.j2  INSTALL.rst  molecule.yml  playbook.yml  tests
+
+* Since `Docker`_ is the default :ref:`driver`, we find a ``Dockerfile.j2``
+  `Jinja2`_ template file in place. Molecule will use this file to build a
+  docker image to test your role against.
+
+* ``INSTALL.rst`` contains instructions on what additional software or setup
+  steps you will need to take in order to allow Molecule to successfully
+  interface with the driver.
+
+* ``molecule.yml`` is the central configuration entrypoint for Molecule. With
+  this file, you can configure each tool that Molecule will employ when testing
+  your role.
+
+* ``playbook.yml`` is the playbook file that contains the call site for your
+  role. Molecule will invoke this playbook with ``ansible-playbook`` and run it
+  against an instance created by the driver.
+
+* ``tests`` is the tests folder created because Molecule uses `testinfra`_ as
+  the default :ref:`verifier`. This allows you to write specific tests against the
+  state of the container after your role has finished executing. Other verifier
+  tools are available.
+
+.. _Docker: https://docs.docker.com/
```

Comment:
This reference is declared 3 times in this document. I think one should be enough for it to work.